.travis.yml

```diff
@@ -77,7 +80,22 @@ matrix:
       services:
         - docker
       script:
-        - "./build/travis/job2_AppImage/build.sh --32bit --upload-branches $APPIMAGE_UPLOAD_BRANCHES"
+        - "./build/travis/job2_AppImage/build.sh --i686 --upload-branches $APPIMAGE_UPLOAD_BRANCHES"
+
+    # 4th parallel build job - portable Linux AppImage armhf build on Debian crosscompiler
+    - env: "JOB=AppImage_armhf"
+      addons:
+        apt:
+          packages:
+          - bsdtar
+          - curl
+          - zsync
+      services:
+        - docker
+      before_script:
+        - "sudo ./build/travis/job2_AppImage/set-binfmt-misc.sh"
+      script:
+        - "./build/travis/job2_AppImage/build.sh --armhf --upload-branches $APPIMAGE_UPLOAD_BRANCHES"
```

Comment:
Fine by me. Here ways to do [containsString in bash](http://stackoverflow.com/questions/229551/string-contains-in-bash). You could do this:
<pre>[[ "$APPIMAGE_BUILD_ARCHS" == *armhf* ]] && ./build</pre>
However, that would cause Travis to fail due to non-zero exit status from the test if it didn't contain "armhf". Any of these would protect it:
<pre>[[ "$APPIMAGE_BUILD_ARCHS" == *armhf* ]] && ./build || true</pre>
<pre>[[ "$APPIMAGE_BUILD_ARCHS" != *armhf* ]] || ./build</pre>
<pre>if [[ "$APPIMAGE_BUILD_ARCHS" == *armhf* ]]; then
  ./build
fi</pre>
(The double square brackets are needed to use bash's built-in test function, rather than the POSIX test function which doesn't have wildcards.)